server/src/com/cloud/server/StatsCollector.java

```diff
@@ -310,8 +326,17 @@ private void init(Map<String, String> configs) {
             _executor.scheduleAtFixedRate(new VmDiskStatsTask(), vmDiskStatsInterval, vmDiskStatsInterval, TimeUnit.SECONDS);
         }
 
+        if (vmNetworkStatsInterval > 0) {
+            if (vmNetworkStatsInterval < 300)
```

Comment:
So why is this hardcoded to be at least 300? Shouldn't we at least throw a info log message here telling the admin that we do this?